lib/jellyfish_web/api_spec/component/rtsp.ex

```diff
@@ -4,43 +4,75 @@ defmodule JellyfishWeb.ApiSpec.Component.RTSP do
   require OpenApiSpex
   alias OpenApiSpex.Schema
 
+  defmodule Metadata do
+    @moduledoc false
+
+    require OpenApiSpex
+
+    OpenApiSpex.schema(%{
+      title: "ComponentMetadataRTSP",
+      description: "Metadata specific to the RTSP component",
+      type: :object,
+      properties: %{}
```

Comment:
It probably won't. However since this is a valid API spec and it seems that quite often we have a case when we want to have an empty schema, maybe we should fix it in the python generator.

Reply:
Update: we have verified that (thanks to the discriminator) the Python generator has no issues with this, hooray